b1ddi/resource_ipam_address.go

```diff
@@ -27,14 +28,21 @@ func resourceIpamsvcAddress() *schema.Resource {
 			StateContext: schema.ImportStatePassthroughContext,
 		},
 		Schema: map[string]*schema.Schema{
-
+			// Defined for Terraform usage. This field won't be used in the API call
+			"next_available_id": {
```

Comment:
Can you also please add a `ForceNew: true` to this attribute? This is so that, if the user updates this field, it should destroy the old resource and create a new one.

Reply:
Ok. Will do